charts/cp-kafka/templates/rbac.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
```

Comment:
I realize that this is from the original PR, but I'm not sure this should be a ClusterRole. But it's probably fine.

Reply:
The same comment applies to the rest of the roles here. Maybe the `nodes` resources are _only_ available to ClusterRoles because they're not namespaced?